tests/providers/dropbox/test_provider.py

```diff
@@ -1,40 +1,30 @@
-import pytest
-
 import io
 import json
-from http import client
+from http import HTTPStatus
 
+import pytest
 import aiohttpretty
 
 from waterbutler.core import streams
-from waterbutler.core import metadata
-from waterbutler.core import exceptions
 from waterbutler.core.path import WaterButlerPath
+from waterbutler.core import metadata as core_metadata
```

Comment:
* Use alias for imports to handle ambiguity in code (several occurrences in other files as well).